docs/Reference/Pantheon-API-Methods.md

```diff
@@ -739,7 +781,21 @@ Returns the account balance of the specified address.
       "result" : "0x0"
     }
     ```
-
+    
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{ "query": "{ account ( address: \"0xfe3b557e8fb62b89f4916b721be55ceb828dbd73\") { balance } }"}' http://localhost:8547/graphql
```

Comment:
Is the network these queries are executed against specified anywhere?

Reply:
No.  A lot of them are against a private network - I've now realised we should have examples from a public network and specify which one so they can be used as is.  Have raised https://pegasys1.atlassian.net/browse/PAN-2714 for this task.